drivers/dma/dma_intel_adsp_hda.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_DMA_DMA_INTEL_ADSP_HDA_COMMON_H_
+#define ZEPHYR_DRIVERS_DMA_DMA_INTEL_ADSP_HDA_COMMON_H_
```

Comment:
DMA_DMA?

Reply:
that was there before dma/dma_intel_adsp_hda_common.h